### PR TITLE
[Windows] Move choco installation list into toolset

### PR DESCRIPTION
**Pull request by nikita-bykov**
# Description
Move the list of tools that can be installed in one line `Choco-Install` to the toolset and make common installation script `Install-CommonUtils.ps1`, like we have for Ubuntu and macOS.

**The list of tools:**

- Install-7zip.ps1
- Install-AzCopy.ps1
- Install-Bicep.ps1
- Install-Cmake.ps1
- Install-GitVersion.ps1
- Install-InnoSetup.ps1
- Install-Jq.ps1
- Install-Julia.ps1
- Install-Nuget.ps1
- Install-OpenSSL.ps1
- Install-Packer.ps1
- Install-Perl.ps1
- Install-Pulumi.ps1
- Install-Svn.ps1
- Install-Swig.ps1
- Install-VSWere.ps1

#### Related issue:
[Move choco installation list into toolset #2374
](https://github.com/actions/virtual-environments-internal/issues/2374)
## Check list
- [x] Related issue / work item is attached
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/Installers/Install-Bazel.ps1 [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-
+################################################################################
+##  File:  Install-Bazel.ps1
+##  Desc:  Install Bazel and Bazelisk (A user-friendly launcher for Bazel)
+################################################################################
+
+Choco-Install -PackageName bazel
+
+npm install -g @bazel/bazelisk
+
```
Comment (AlenaSviridenko) [inline images/win/scripts/Installers/Install-Bazel.ps1:9]: What was changed here?

**Review by kassannk — APPROVED** [review]

File: images/win/toolsets/toolset-2019.json [hunk 2]
```diff
@@ -374,5 +374,31 @@
...
+            { "name": "7zip.install"},
+            { "name": "azcopy10"},
+            { "name": "Bicep"},
+            { "name": "gitversion.portable"},
+            { "name": "innosetup"},
+            { "name": "jq"},
+            { "name": "NuGet.CommandLine"},
+            { "name": "openssl.light"},
+            { "name": "packer"},
+            { "name": "strawberryperl"},
```
Comment (miketimofeev) [inline images/win/toolsets/toolset-2019.json:389]: There was an issue with strawberry perl, openssl, and git in the PATH. Need to make sure that nothing is broken after these changes
https://github.com/actions/virtual-environments/pull/548

File: images/win/scripts/Installers/Install-CommonUtils.ps1 [hunk 3]
```diff
@@ -0,0 +1,13 @@
+$commonPackages = (Get-ToolsetContent).choco.common_packages
+
+foreach ($package in $commonPackages)
+{
+    Write-Host "Installing ${package.name}"
```
Comment (AlenaSviridenko) [inline images/win/scripts/Installers/Install-CommonUtils.ps1:5]: I don't think this is necessary, choco outputs the package name it is currently installing.

**Review by dmitry-shibanov — APPROVED** [review]

File: images/win/scripts/Installers/Install-CommonUtils.ps1 [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    if ($package.args) {
```
Comment (miketimofeev) [inline images/win/scripts/Installers/Install-CommonUtils.ps1:6]: Maybe we can get rid of it and pass $package.args anyway. Choco-install will skip them if the variable are empty

**Review by vsafonkin — DISMISSED** [review]

File: images/win/toolsets/toolset-2019.json (hunk 2 above)
Comment (nikita-bykov) [reply images/win/toolsets/toolset-2019.json:389]: Compared the PATH in current image and after my changes, the order of directories for strawberry perl, openssl, and git is the same

**Review by MaksimZhukov — APPROVED** [review]

File: images/win/toolsets/toolset-2019.json (hunk 2 above)
Comment (miketimofeev) [reply images/win/toolsets/toolset-2019.json:389]: Thanks for the update!

**Review by dmitry-shibanov — APPROVED** [review]

**Review by AlenaSviridenko — APPROVED** [review]

File: images/win/toolsets/toolset-2019.json (hunk 2 above)
Comment (miketimofeev) [reply images/win/toolsets/toolset-2019.json:389]: The PATH has actually changed a bit, but it looks like nothing critical:
image
All the applications are in the same places judging by `Get-Command -CommandType Application`

**Review by miketimofeev — DISMISSED** [review]

**Review by maxim-lobanov — DISMISSED** [review]

**Review by maxim-lobanov — APPROVED** [review]

**Pull request merged**